docs/user/faq/troubleshooting.md

```diff
@@ -2,106 +2,118 @@
-
+If you have entered the correct recovery token, you'll then be prompted to enter a new admin password.
+<br /><br />
 <p align="center">
     <img src="../../../img/sign_in_image.png"/>
 </p>
-
-Remember using a password with the next properties:
-
-- Minimum 8 characters
-- At least one Cap letter
-- At least one number
-
-Once you filled the field and press the register button, dappnode will provide a recovery token.
-
+Once you've filled out those fields and pressed 'Register', your DAppNode will once again provide you a recovery token. <b>Do not forget to write down and save this one!</b>
 <p align="center">
     <img src="../../../img/recovery_token.png"/>
 </p>
 
-We write down and save that token and we accept we have copied our recovery token. We log in with our new credentials and we have access to our DAppNode.
-
-## How to recover your recovery token and recover admin access
+</div>
+  </div>
+</details>
 
-In case we have lost both the password and the recovery token, we need to access our DAppNode: SSH into the DAppNode host or plug a keyboard.
-
-Once we are in our DAppNodeMachine, we type the next command which shows us the recovery token:
+<details>
+  <summary>What do I do if I lost the password AND my token?!</summary>
+  <div>
+    <div>
+In the case where you have lost both the password and the recovery token, we will need either SSH or local access to the DAppNode machine.
+<br /><br />
+Once we are connected to our DAppNode, we type the next command which prints the recovery token to the terminal:
 
 ```
 cat /usr/src/dappnode/DNCORE/admin-recovery-token.txt ; echo
 ```
 
-The command does the next:
+The command does the following:
+
+- We read the admin-recover-token.txt file where the token is saved.
+- The <code>; echo</code> is used to make it easier to read and copy the token.
 
-- The admin-recover-token.txt is where the token is saved.
-- The <code>; echo</code> is used to make easier the read of the token.
+After inserting the command above, we obtain the recovery token and use that one to recover our admin access.
 
-After inserting the command we obtain the recovery token we have to use to reset our password.
-On the above section, How to access to my DAppNode using the recovery token you can reset your password with this recovery token.
+In the question above you can check out how to use your token to regain access.
+</div>
+  </div>
+</details>
 
-## I need to restore the system without losing any data
+<details>
+  <summary>How do I force update my DAppNode without losing data?</summary>
+  <div>
+    <div>
+    If you are experiencing an issue or just want to make sure you are running the latest DAppNode version, execute this command below in the DAppNode terminal. 
+    <br /><br />
+    This will update the core packages to the latest versions without erasing any data from your DAppNode.
```

Comment:
```suggestion
    This will update the core packages to the latest versions without erasing any data from your DAppNode.
    ⚠️ This will clear any EXTRA_OPTS fields, but no chain data, keys, or other configurations will be affected
```